Bibliotheque nationale de France.js

```diff
@@ -43,6 +43,8 @@ var BnfClass = function () {
 	*/
 	
 	function getCreatorType(aut) {
+		// To avoid an error on certain pages where the item is not well catalogued
+		if (typeof aut['4'] === 'undefined') return undefined;
```

Comment:
Can just do `if (aut['4'] === undefined) return undefined;` these days.

Reply:
What do you think of [this answer](https://stackoverflow.com/a/3550319)?

Reply:
`undefined` has been immutable for many years, so when comparing against a property it's fine to do `foo.bar === undefined`. The only time you need to use `typeof` is if a variable may not be defined (e.g., for feature detection), but that's not relevant for a property.

(This is explained well in [another answer](https://stackoverflow.com/a/22053469) on that page.

Reply:
Sorry I deleted my comment before I saw your answer, after reading that answer on SO